td-shim/src/lib.rs

```diff
@@ -6,21 +6,55 @@
 
 use scroll::{Pread, Pwrite};
 
-use td_uefi_pi::pi;
+use td_uefi_pi::{pi, pi::guid};
 
 pub mod acpi;
 pub mod event_log;
 pub mod fv;
 pub mod reset_vector;
 pub mod secure_boot;
 
-pub const TD_ACPI_TABLE_HOB_GUID: [u8; 16] = [
-    0x70, 0x58, 0x0c, 0x6a, 0xed, 0xd4, 0xf4, 0x44, 0xa1, 0x35, 0xdd, 0x23, 0x8b, 0x6f, 0xc, 0x8d,
-];
-
-pub const TD_KERNEL_INFO_HOB_GUID: [u8; 16] = [
-    0x12, 0xa4, 0x6f, 0xb9, 0x1f, 0x46, 0xe3, 0x4b, 0x8c, 0xd, 0xad, 0x80, 0x5a, 0x49, 0x7a, 0xc0,
-];
+pub const TD_ACPI_TABLE_HOB_GUID: guid::Guid = guid::Guid::from_fields(
```

Comment:
Could you please help to add documentation for related sections in the td-shim spec?

Reply:
I added some references to the spec